src/main/webapp/courseLoader.js

```diff
@@ -412,6 +412,7 @@
       ratingProf: document.getElementById("rating-prof").value,
       hours: document.getElementById("hours").value,
       difficulty: document.getElementById("difficulty").value,
+      ID: await verify(),
```

Comment:
I think this might turn ID into a promise here until this resolves or something. Not sure how await inside of objects works.

Just be aware of that if this has weird issues.